node/network/bitfield-distribution/src/lib.rs

```diff
@@ -0,0 +1,439 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can rerelay_message it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is relay_messaged in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
```

Comment:
Pretty sure the incidents of `relay_message` in these paragraphs are from over-enthusiastically applied search-replace, not the intended message.